src/App.css

```diff
@@ -36,3 +37,14 @@
     transform: rotate(360deg);
   }
 }
+span {
+  font-weight: bold;
+}
+p {
+  background-color: orange;
+  width: 100%;
+  height: 250px;
+}
+img {
+  height: 70%;
+}
```

Comment:
These CSS selectors affect all the `span`, `p`, and `img` in the project. They should be related to the component only.